packages/zk-transfer-web/src/libs/types.ts

```diff
@@ -1,5 +1,38 @@
 import { Map, List } from "immutable";
 
+export type BM = any;
+
+type Password = string;
+
+export type SignerEth = {};
+
+export type EthAddress = string;
+
+type SignerMap = {
```

Comment:
Thanks for reverse engineering this.

There's also the beginning of API documentation at
https://democracy.js.org/jsdocs/module-wallet.html#.prepareSignerEth

Reply:
Okay!